wrapper/src/main/java/com/splunk/support/lambda/configuration/SplunkConfiguration.java

```diff
@@ -51,17 +43,8 @@
           "io.opentelemetry.sdk.extension.aws.resource.EksResource");
 
   public static void configure() {
-    setDefaultValues();
-    Config.internalInitializeConfig(
-        new ConfigBuilder().readEnvironmentVariables().readSystemProperties().build());
+    setDefaults();
     configureOtelLogging();
-
-    OpenTelemetrySdk.builder()
```

Comment:
We need to call the Autoconfiguration method here or nothing will get configured.

Reply:
Done. I assumed it's automagically initialized by the way.

Reply:
yes, doing it at configuration time, rather than in the hot path during lambda execution seems much better. And, making it explicit makes the code much easier to reason about.

Reply:
Eventually I had to remove this, as apparently parent classes from AWS instrumentation library already call initialized() and double call ends up with a nasty exception.

Reply:
Wow... and that's all because of a missing `flush()` in `TracerProvider` 😂